tests/helpers.py

```diff
@@ -189,7 +189,11 @@ def create_storage_class(
     sc_data['parameters']['csi.storage.k8s.io/provisioner-secret-name'] = secret_name
     sc_data['parameters']['csi.storage.k8s.io/provisioner-secret-namespace'] = defaults.ROOK_CLUSTER_NAMESPACE
 
-    sc_data['parameters']['monitors'] = mons
+    if interface_type == constants.CEPHBLOCKPOOL:
+        sc_data['parameters']['clusterID'] = defaults.ROOK_CLUSTER_NAMESPACE
```

Comment:
If this cluster namespace is going to be changed by at some point from custom config file then I would rather access this CLUSTER_NAMESPACE from config.ENV_DATA which refer all the time to one currently used for running cluster.  I won't block PR on this, but just be aware of this and maybe change in new PR if it suppose to reflect the cluster namespace of current cluster.

Reply:
+1